packages/runtime/src/modules/RequestModule.ts

```diff
@@ -114,6 +114,29 @@ export class RequestModule extends RuntimeModule {
             return;
         }
 
+        const otherRequestsThatWouldLeadToARelationship = (
+            await services.consumptionServices.incomingRequests.getRequests({
+                query: {
+                    "source.type": "RelationshipTemplate",
+                    status: [LocalRequestStatus.Open, LocalRequestStatus.DecisionRequired, LocalRequestStatus.ManualDecisionRequired],
```

Comment:
There's the very similar case above where the newly loaded template is the template with existing requests. There the status `Decided` is included as well which I prefer.

Reply:
Also in the case above `Expired` is not excluded, this should be fixed

Reply:
I will not add Decided because

1. if the decision is accepted there would be a Relationship
2. if the decision is rejected I should be able to receive the Request

Reply:
I added the filter for decided.

I'd like to end the decision about the above check as it doesn't correlates with my PR.